external/kafka-assembly/pom.xml

```diff
@@ -43,6 +43,11 @@
     </dependency>
     <dependency>
       <groupId>org.apache.spark</groupId>
+      <artifactId>spark-streaming-kafka-newapi_${scala.binary.version}</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
```

Comment:
Mark, the jar  (spark-streaming-kafka-assembly*.jar)  produced by this assembly works only with kafka 0.9 brokers/servers. I have tested it.

So if i want to use Spark 2.0,  and continue to run my existing spark-streaming-kakfka-app (uses 'older' spark-streaming-kakfa api) because i cannot upgrade  kafka brokers/servers to 0.9, there is no spark-streaming-kafka-assembly jar provided by spark 2.0 for that?

I think we need to provide for above, which means we need produce a 'spark-streaming-kafka-newapi-assembly*.jar' and a 'spark-streaming-kafka-assembly*.jar' .  I am not sure if it is a best practice to produce 2 diff assembly.jar from a single pom.xml if there is a way, else just have a spark-kafka-assembly-newapi